### ISSUE TITLE
Error handler not showing backtraces for linked exceptions

### ISSUE THREAD

**Issue by tvdavid** [yiisoft/yii2#4224]
I have a twig layout, which calls a widget, which happened to cause a UnknownPropertyException. 

The backtrace I get doesn't tell me the reason for the exception though:

```
Twig_Error_Runtime
An exception has been thrown during the rendering of a template ("Getting unknown property: app\models\editable\GuestCast::name") in "../node/_form.twig" at line 5.

Caused by: Unknown Property – yii\base\UnknownPropertyException
Getting unknown property: app\models\editable\GuestCast::name

in /web/vendor/yiisoft/yii2/base/Component.php at line 142

1. in /web/vendor/twig/twig/lib/Twig/Template.php
2. in /web/vendor/twig/twig/lib/Twig/Template.php
3. in /Twig/cache/48/5e/935ded6ceee4a4272e44fb5d66ac1da5e99ea32a790124d34f837dbb6779.php
4. in /web/vendor/twig/twig/lib/Twig/Template.php
5. in /web/vendor/twig/twig/lib/Twig/Template.php
6. in /Twig/cache/86/04/7e1d5fcf7908a660ce8d5e69f3a42c96063cc0d87f36800756c463dc4656.php
7. in /web/vendor/twig/twig/lib/Twig/Template.php
8. in /web/vendor/twig/twig/lib/Twig/Template.php
9. in /Twig/cache/8d/a6/7f4484e5613661aaadc5b3c9f8fa7bbed68d0aefb23545f6c75f1e71b920.php
10. in /web/vendor/twig/twig/lib/Twig/Template.php
11. in /web/vendor/twig/twig/lib/Twig/Template.php
12. in /web/vendor/twig/twig/lib/Twig/Template.php
13. in /web/vendor/twig/twig/lib/Twig/Environment.php
14. in /web/vendor/yiisoft/yii2-twig/ViewRenderer.php
15. in /web/vendor/yiisoft/yii2/base/View.php
16. in /web/vendor/yiisoft/yii2/base/View.php
17. in /web/vendor/yiisoft/yii2/base/Controller.php
18. in /web/controllers/base/NodeController.php 
19. in /web/controllers/GuestcastController.php
```

Apparently it's swallowed by twig, because if I comment out these lines in vendor/twig/twig/lib/Twig/Template.php

```
            //} catch (Exception $e) {
            //    throw new Twig_Error_Runtime(sprintf('An exception has been thrown during the rendering of a template ("%s").', $e->getMessage()), -1, $template->getTemplateName(), $e);

        //} catch (Exception $e) {
        //    throw new Twig_Error_Runtime(sprintf('An exception has been thrown during the rendering of a template ("%s").', $e->getMessage()), -1, $this->getTemplateName(), $e);
```

.. I get  much more useful backtrace, leading to the cause of the problem.

```
Unknown Property – yii\base\UnknownPropertyException
Getting unknown property: app\models\editable\GuestCast::name

1. in /web/vendor/yiisoft/yii2/base/Component.php
2. in /web/vendor/yiisoft/yii2/db/BaseActiveRecord.php – yii\base\Component::__get('name') 
3. in /web/models/base/ActiveRecord.php – yii\db\BaseActiveRecord::__get('name') 
4. in /web/models/base/Node.php – app\models\base\ActiveRecord::getSlug() 
```

**Comment by RSalo** [comment]
Issue is actual for now?

**Comment by samdark** [comment]
Twig part of it isn't. See https://github.com/yiisoft/yii2-twig/issues/25

Yii part of it is still is: traces for previous exceptions aren't displayed.

**Comment by samdark** [comment]
It's good to have but it doesn't seem to bother anyone much nowadays.

**Comment by samdark** [comment]
Won't be implemented in 2.0.
